torch/csrc/autograd/functions/accumulate_grad.cpp

```diff
@@ -88,6 +92,11 @@ variable_list AccumulateGrad::apply_with_saved(
       });
   saved.after(variable_copy);
   saved.after(grad_copy);
+
+  for (const auto& hook : tensor_post_acc_grad_hooks()) {
+    (*hook)(variable);
+  }
+
```

Comment:
We don't actually ever reach here because we would've errored during the compiled_args part?

Reply:
Yea...but this is where the code should go if it worked, right?

Reply:
Removing to minimize confusion